src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -203,7 +203,8 @@ def default_api_version(self):
         # Full MultiAPI support is not done in AppService, the line below is merely
         # to have commands show up in the hybrid profile which happens to have the latest
         # API versions
-        ResourceType.MGMT_APPSERVICE: '2018-02-01'
+        ResourceType.MGMT_APPSERVICE: '2018-02-01',
+        ResourceType.MGMT_EVENTHUB: '2017-04-01'
```

Comment:
The title is `for profile 2019-03-01-hybrid`, but added `2017-03-01` instead? Any context?

Reply:
2019-03-01-hybrid is the target profile, which will be used by Azure Stack. I think you mean is 2017-04-01? 2017-04-01 is the api version for eventhub module. They are two different concept.

Actually what I do is adding eventhub support with api version 2017-04-01 for profile 2019-03-01-hybrid. In this way, when cloud users use profile 2019-03-01-hybrid, cli will use api version 2017-04-01 for eventhub commands.